scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -93,6 +93,9 @@
       <div>
         <strong>Local (arch.)</strong>: {{ variant.local_obs_old or 0 }} obs.
       </div>
+      {% if variant.mitomap_associated_diseases and variant.mitomap_associated_diseases != "." %}
```

Comment:
Yes, it would be nice not to save it at all in the database if it doesn't have valid data